### PR TITLE
[INLONG-810] Sort Module Support store data to ApacheDoris 

### LINKED ISSUE #1821: [INLONG-810] Sort Module Support store data to ApacheDoris
### Description

InLong only support store data to Clickhouse now, it's better to support Apache Doris.

JIRA link - [INLONG-810](https://issues.apache.org/jira/projects/INLONG/issues/INLONG-810?filter=allopenissues) 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by huzk8**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #1821 
### Motivation

add sort support doris sink 

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes)
  - If yes, how is the feature documented? ( docs )
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/flink/doris/TestDorisSink.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+		final DataSource<Tuple2<Boolean, Row>> data = env.fromElements(
+				new Tuple2<Boolean, Row>(true, Row.of(1, "pengpeng", 13)),
+				new Tuple2<Boolean, Row>(true, Row.of(2, "xingxing", 2))
+		);
+
+
+		DorisSinkOptions dorisSinkOptions = new DorisSinkOptionsBuilder()
+				.setFeNodes("10.93.6.6:8030")
```
Comment (dockerzhang) [inline inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/flink/doris/TestDorisSink.java:46]: it's  a reveal address？

**Comment by dockerzhang** [comment]
```
[INFO] 开始检查……
Error:  /home/runner/work/incubator-inlong/incubator-inlong/inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/sink/DorisSinkInfo.java:77: 'METHOD_DEF' 前只应有一个空行。 [EmptyLineSeparator]
检查完成。
[INFO] There is 1 error reported by Checkstyle 8.29 with codestyle/checkstyle.xml ruleset.
Error:  src/main/java/org/apache/inlong/sort/protocol/sink/DorisSinkInfo.java:[77] (whitespace) EmptyLineSeparator: 'METHOD_DEF' 前只应有一个空行。
```
@huzk8  please check this error, thanks.

**Comment by ifndef-SleePy** [comment]
Hi @huzk8 , thanks for contributing!
It's nice to support Doris sink for INLONG project. It looks good to me although I'm not an expert of Doris. I think it's fine to merge the PR first and to tune it later if necessary.
So +1 for me.

**Review by dockerzhang — APPROVED** [review]
LGTM

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/DorisRespond.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package org.apache.inlong.sort.flink.doris.load;
+
+/**
+ * @program: incubator-inlong
```
Comment (gosonzhang) [inline inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/DorisRespond.java]: After the class function is annotated, the author and creation time need to be removed

**Review by gosonzhang — CHANGES_REQUESTED** [review]

File: inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/BeInfoResponse.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package org.apache.inlong.sort.flink.doris.load;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @program: incubator-inlong
+ * @author: huzekang
```
Comment (gosonzhang) [inline inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/BeInfoResponse.java]: the same problem

**Comment by gosonzhang** [comment]
@huzk8,  there are several review comments that you need to modify

**Comment by dockerzhang** [comment]
@huzk8 do you plan to fix the last problems according to the review comments？ We will release the next version in two weeks, and we hope this PR can be merged.

**Comment by huzk8** [comment]
> @huzk8, there are several review comments that you need to modify

the `inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/doris/load/DorisRespond.java` is already modify in commit #e3fe9df

**Comment by huzk8** [comment]
> @huzk8, there are several review comments that you need to modify

could you tell me more detail about which file need to modify

**Review by gosonzhang — APPROVED** [review]
+1

**Comment by dockerzhang** [comment]
@huzk8 thanks for this contribution.

**Pull request merged**